system-helper/org.freedesktop.Flatpak.rules.in

```diff
@@ -7,6 +7,11 @@ polkit.addRule(function(action, subject) {
         subject.active == true && subject.local == true &&
         subject.isInGroup("@privileged_group@")) {
             return polkit.Result.YES;
+    } else if (action.id == "org.freedesktop.Flatpak.app-install" &&
+               subject.active == true && subject.local == true &&
+               action.lookup("parental-controls-repo-installation-allowed") === "1" &&
+               action.lookup("parental-controls-is-appropriate") === "1") {
+            return polkit.Result.YES;
```

Comment:
I'm not sure about this, you're not applying an extra checks here. Instead you are opening a new hole for non-admins to install system-wide apps which they were not allowed to before.

Reply:
Sure, but the policy toggle for that is [disabled by default](https://gitlab.freedesktop.org/pwithnall/malcontent/blob/126190a6a9d74eba1f4390e6429e4145bc9e5ab1/accounts-service/com.endlessm.ParentalControls.AppFilter.xml#L90).

Reply:
I don't understand. Why not check `subject.isInGroup("@privileged_group@")` in this set of conditions?

Reply:
This code has significantly changed in the new version of the PR.